tools/nni_cmd/launcher.py

```diff
@@ -371,8 +371,9 @@ def launch_experiment(args, experiment_config, mode, config_file_name, experimen
             exit(1)
     log_dir = experiment_config['logDir'] if experiment_config.get('logDir') else None
     log_level = experiment_config['logLevel'] if experiment_config.get('logLevel') else None
-    if log_level not in ['trace', 'debug'] and (args.debug or experiment_config.get('debug') is True):
-        log_level = 'debug'
+    if mode != 'view':
```

Comment:
is it necessary to check whether mode is 'view' or not?

Reply:
Add debug mode in experiment view.